.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+Status: **Ready for Review** or **Open for Visibility**
+Reviewers: GitHub @<name>s of all reviewers
+
+Linked PRs: <links to corresponding PRs, optional>
+
+## Changes
+- <changes here>
+
+## Todos:
+- [ ] +1
```

Comment:
That's right. I have seen things like "UI dev +1", "Eng +1", and "Design +1" to keep track of who still has to provide their thumbs up